Find a functionally equivalent code:android.os.Bundle.getFloatArray:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the value associated with the given key, or null if
 * no mapping of the desired type exists for the given key or a null
 * value is explicitly associated with the key.
 *
 * @param key a String, or null
 * @return a float[] value, or null
 */

@@ -1,13 +1,3 @@
 {
-    unparcel();
-    Object o = mMap.get(key);
-    if (o == null) {
-        return null;
-    }
-    try {
-        return (float[]) o;
-    } catch (ClassCastException e) {
-        typeWarning(key, o, "float[]", e);
-        return null;
-    }
+    return super.getFloatArray(key);
 }

